ecmd-core/capi/ecmdClientCapi.H

```diff
@@ -4287,6 +4287,23 @@ uint32_t spiWrite(const ecmdChipTarget & i_target, uint32_t i_engine, uint32_t i
 #endif /* ifndef ECMD_REMOVE_SPI_FUNCTIONS (this comment has to be here for makedll.pl to work */
 //@}
 
+/** @name Configuration Functions */
+//@{
+/**
+ @brief Set the value of a supported Configuration Setting like for example scanmode
+ @param i_target Struct that specifies the target to operate on (see target depth and states below)
+ @param i_name Name of setting as defined by eCMD Api
+ @param i_configData Structure containing the valid data
```

Comment:
Need to add something here for i_mode.

 @param i_mode setconfig mode bit-field using ECMD_SETCONFIG_MODE* defines
 
 You could also add a NOTE section at the end here to provide a little more information on what the behavior is for the different modes.